networking/egress-ips.html.md

```diff
@@ -9,10 +9,10 @@ date: 2025-10-02
 ## Overview
 
 - By default, outbound (egress) IPs from Fly Machines are **unstable** and may change.
-- You can allocate **static egress IPs** per machine (both IPv4 and IPv6) via `fly machine egress-ip`.
-- Static egress IPs come with trade-offs: cost, binding to machine lifecycle, and deployment quirks.
-- A common workaround is to front outbound traffic through a **proxy** app that _does_ have static egress IPs.
-- App-scoped egress IPs are in development and may simplify this in the future.
+- You can allocate **static egress IPs** for an app (both IPv4 and IPv6) via `fly ips allocate-egress`.
+- App-scoped Static egress IPs are per-region: you need one for every region where you have machines.
+- Static egress IPs come with trade-offs: cost, concurrent machines / connections.
```

Comment:
Can you clarify this a bit more? Is the trade-off that they will hit limits for concurrent machines and connections? If that is the case maybe say something like: "Static egress IPs come with trade-offs: they cost more, and limit how many machines or connections you can run at once."